po/pt-BR.po

```diff
@@ -7057,37 +7095,29 @@ msgid ""
 "arm, `half` is bound to the value inside the `Ok` variant. In the second arm,\n"
 "`msg` is bound to the error message."
 msgstr ""
-"Aqui usamos a verificação de correspondência para _desestruturar_ o valor contido em `Result`. Na "
-"primeira\n"
-"verificação de correspondência, `metade` está vinculado ao valor dentro da variante `Ok`. Na "
-"segunda,\n"
+"Aqui usamos a verificação de correspondência para _desestruturar_ o valor contido em `Result`. Na primeira\n"
+"verificação de correspondência, `metade` está vinculado ao valor dentro da variante `Ok`. Na segunda,\n"
 "`msg` está vinculado à mensagem de erro."
 
 #: src/pattern-matching/destructuring-enums.md:36
-#, fuzzy
 msgid ""
 "* The `if`/`else` expression is returning an enum that is later unpacked with a `match`.\n"
-"* You can try adding a third variant to the enum definition and displaying the errors when running "
-"the code. Point out the places where your code is now inexhaustive and how the compiler tries to "
-"give you hints."
+"* You can try adding a third variant to the enum definition and displaying the errors when running the code. Point out the places where your code is now "
+"inexhaustive and how the compiler tries to give you hints."
 msgstr ""
-"Pontos chave:\n"
-"* A expressão `if`/`else` está retornando um enum que é posteriormente descompactado com um "
-"`match`.\n"
-"* Você pode tentar adicionar uma terceira variante à definição de Enum e exibir os erros ao "
-"executar o código. Aponte os lugares onde seu código agora é \"não exaustivo\" e como o compilador "
-"tenta lhe dar dicas."
+"* A expressão `if`/`else` está retornando um `enum` que é posteriormente descompactado com um `match`.\n"
+"* Você pode tentar adicionar uma terceira variante à definição de Enum e exibir os erros ao executar o código. Aponte os lugares onde seu código agora é \"não "
+"exaustivo\" e como o compilador tenta lhe dar dicas."
 
 #: src/pattern-matching/destructuring-structs.md:1
 msgid "# Destructuring Structs"
-msgstr "# Desestruturando estruturas (Structs)"
+msgstr "# Desestruturando Estruturas"
```

Comment:
Likewise, perhaps you can use _Structs_ as it's used in the pt-BR Rust Book: https://rust-br.github.io/rust-book-pt-br/ch05-00-structs.html